.secrets.baseline

```diff
@@ -45,7 +45,38 @@
       "name": "StripeDetector"
     }
   ],
-  "results": {},
+  "results": {
+    "lambda/health_package/conftest.py": [
+      {
+        "hashed_secret": "5231904b911e0db291f01dfe9c92473f2a3c1b49",
+        "is_secret": false,
+        "is_verified": false,
+        "line_number": 253,
+        "type": "Hex High Entropy String"
+      },
+      {
+        "hashed_secret": "4fe6f9a3d47a49217e58dcf9f51d4e715ff3b227",
+        "is_secret": false,
+        "is_verified": false,
+        "line_number": 254,
+        "type": "Hex High Entropy String"
+      },
+      {
+        "hashed_secret": "92ecf3c81d61770ca707bf32cfbff86417ec76f3",
+        "is_secret": false,
+        "is_verified": false,
+        "line_number": 255,
+        "type": "Hex High Entropy String"
+      },
+      {
+        "hashed_secret": "01be741099a377ecf0810426de0ec62ada610a4c",
+        "is_secret": false,
+        "is_verified": false,
+        "line_number": 307,
+        "type": "Base64 High Entropy String"
+      }
+    ]
+  },
```

Comment:
:ok_hand:

Reply:
> Tests fail for me, using `make test` in a fresh venv

Try now. (I closed the PR completely by accident, pressed the wrong button) 🙄